src/tests/ftest/deployment/target_failure.py

```diff
@@ -144,6 +144,10 @@ def verify_failure_with_protection(self, ior_namespace):
         self.pool.measure_rebuild_time(
             operation="Reintegrate rank 0 -> target 1", interval=5)
 
+        # Container status doesn't automatically go back to healthy after reintegrate. We
+        # need to manually set it.
+        self.container.set_prop(prop='status', value="healthy")
```

Comment:
(pylint-no-member) Instance of 'TestContainer' has no 'set_prop' member; maybe 'get_prop'?
(pylint-no-member) Instance of 'list' has no 'set_prop' member